updater/lib/dependabot/setup.rb

```diff
@@ -37,7 +37,31 @@
     pub
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = lambda do |event, hint|
+    debugger
+    if !hint[:exception].nil?
+      ExceptionSanitizer.process_sentry(event, hint)
+
+      # TODO integrate these two...
+      # TODO copied from migration guide, I don't understand how this assignment works though?
+      # https://docs.sentry.io/platforms/ruby/migration/#exceptionraven_context
+      if exception = hint[:exception]
+        exception.raven_context.each do |key, value|
```

Comment:
I'll need to look into this... at a minimum, I'll probably want to rename this to `sentry_context`